torch/nn/modules/transformer.py

```diff
@@ -134,20 +150,21 @@ def __init__(self, encoder_layer, num_layers, norm=None):
         self.num_layers = num_layers
         self.norm = norm
 
-    def forward(self, src, mask=None):
+    def forward(self, src, mask=None, key_padding_mask=None):
         r"""Pass the input through the endocder layers in turn.
 
         Args:
             src: the sequnce to the encoder (required).
-            src_mask: the mask for the src sequence (optional).
+            mask: the mask for the src sequence (optional).
+            key_padding_mask: the mask for the src keys per batch (optional).
```

Comment:
For consistency with Transformer, I think we can use "src_key_padding_mask" here, instead of key_padding_mask.